content/en/blog/_posts/2018-12-05-kubeadm-ga-release.md

```diff
@@ -0,0 +1,91 @@
+- Specific cloud provider integrations
+
+Infrastructure provisioning, for example, is left to other SIG Cluster Lifecycle projects, such as the [Cluster API](https://github.com/kubernetes-sigs/cluster-api). Instead, kubeadm covers only the common denominator in every Kubernetes cluster: the [control plane](https://kubernetes.io/docs/concepts/#kubernetes-control-plane). The user may install their preferred networking solution and other add-ons on top of Kubernetes *after* cluster creation.
+
+The core of the kubeadm interface is quite simple: new control plane nodes are created by running `kubeadm init` and worker nodes are joined to the control plane by running `kubeadm join`. Also included are utilities for managing already bootstrapped clusters, such as control plane upgrades and token and certificate renewal.
+
+### What kubeadm's GA release means
+
+General Availability means different things for different projects. For kubeadm, going GA means not only that creating a conformant Kubernetes cluster is now stable, but also that kubeadm is flexible enough to support a wide variety of deployment options.
+
+We now consider kubeadm to have achieved GA-level maturity in each of these important domains:
+
+ * **Stable command-line UX** — The kubeadm CLI conforms to [#5a GA rule of the Kubernetes Deprecation Policy](https://kubernetes.io/docs/reference/using-api/deprecation-policy/#deprecating-a-flag-or-cli), which states that a command or flag that exists in a GA version must be kept for at least 12 months after deprecation.
+ * **Stable underlying implementation** — kubeadm now creates a new Kubernetes cluster using methods that shouldn't change any time soon. The control plane, for example, is run as a set of static Pods, bootstrap tokens are used for the **kubeadm join** flow, and [ComponentConfig](https://github.com/kubernetes/community/blob/master/keps/sig-cluster-lifecycle/0014-20180707-componentconfig-api-types-to-staging.md) is used for configuring the kubelet.
+ * **Configuration file schema** — With the new  **v1beta1** API version, you can now tune almost every part of the cluster declaratively and thus build a "GitOps" flow around kubeadm clusters. In future versions, we plan to graduate the API to version **v1** with minimal changes (and perhaps none).
+ * **The "toolbox" interface of kubeadm** — Also known as **phases**. If you don't want to consume the all of the **kubeadm init** tasks performed, you can perform fine-grained, generic actions with the **kubeadm init phase** command (for example, generating certificates or control plane Static Pod manifests).
+ * **Upgrades between minor versions** — The **kubeadm upgrade** command is now fully GA. It handles control plane upgrades for you, which includes upgrades to etcd, the API Server, the Controller Manager, and the Scheduler. You can seamlessly upgrade your cluster between minor or patch versions (e.g. v1.12.2 -> v1.13.1 or v1.13.1 -> v1.13.3)
```

Comment:
do we want to say anything about `downgrades`?

Reply:
Nope.